feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -127,6 +128,18 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	dc.Interface(i2.GetName()).Replace(t, i2)
 }
 
+//Configure network instance
+func configureNetworkInstance(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	d := &telemetry.Device{}
```

Comment:
I know it's a little verbose, but how about constructing the network instance struct directly?

ni := &telemetry.NetworkInstance{
  Name: ...,
  Type: ...,
  RouterId: ...,
}